warehouse/models/mart/gtfs_quality/fct_daily_rt_feed_guideline_checks.md

```diff
@@ -18,4 +18,7 @@ Here is a list of currently-implemented checks:
 | Service alerts RT feed uses HTTPS | Best Practice Alignment | The service alerts RT feed endpoint uses HTTPS instead of HTTPS to ensure feed integrity.|
 |Vehicle positions RT feed uses HTTPS | Best Practice Alignment | The vehicle positions RT feed endpoint uses HTTPS instead of HTTPS to ensure feed integrity.|
 | Trip updates RT feed uses HTTPS | Best Practice Alignment | The trip updates RT feed endpoint uses HTTPS instead of HTTPS to ensure feed integrity.|
+| Fewer than 1% of requests to Trip updates RT feed result in a protobuf error | Best Practice Alignment | On the given day, fewer than 1% of requests to Trip updates RT feed result in a protobuf error.|
```

Comment:
This is incredibly nitpicky so feel free to disregard but "1% of **requests**" is perhaps not quite accurate here, because I think the denominator you're using is **files**. So if there's a URL where half of requests fail (no file returned) but the protos we do get are 100% parseable, this check will pass.

Reply:
How about: 
"On the given day, fewer than 1% of Trip updates RT feed downloads result in a protobuf error."

Reply:
Yes sounds good!